Dozer/DozerIcons.swift

```diff
@@ -139,7 +153,27 @@ public final class DozerIcons {
         }
     }
 
-    /// Force show all Dozer status bar icons
+    public func showIconAndMenu() {
+        if NSWorkspace.shared.frontmostApplication?.bundleIdentifier != "com.mortennn.Dozer" {
```

Comment:
As a nitpick: I wouldn't hardcode this. [It seems there's a constant for it](https://github.com/Mortennn/Dozer/blob/9705af61301c90b4d91928a957378d9d30c8b295/Dozer/Constants.swift#L37), though personally I'm surprised if the build process or the runtime don't supply it from Info.plist.

Reply:
Good spot!